netty/src/test/java/io/grpc/netty/NettyStreamTestBase.java

```diff
@@ -199,7 +200,7 @@ public void shouldNotBeReadyForDataAfterWritingLargeMessage() throws IOException
   }
 
   protected byte[] smallMessage() {
-    return MESSAGE.getBytes();
+    return MESSAGE.getBytes(US_ASCII);
```

Comment:
Versus what? The only other thing I see is StandardCharsets, which we can't use. I don't see anything lead me to believe that US_ASCII is any more deprecated than UTF_8.